src/Inputs/SelectInput.php

```diff
@@ -23,7 +23,7 @@ class SelectInput extends SelectBox implements IValidationInput
 
 	/**
 	 * @param string|Html|null       $label
-	 * @param string[]|null $items
+	 * @param array<string|int, array<string|int, string>|string>|null $items
```

Comment:
Tedka koukam co znamena ta carka tam? To je nebo? Neslo by pouzit `|`?

Reply:
Já si myslím, že je to takhle dobře. klíče jsou `string|int` případně si myslím, že teoretiky by prošlo i `bool` a hodnota je buď string nebo zase pole.

Reply:
Jo takhle, to je ty pro array. Uz to vidim.

Reply:
jinak čárka dělí tu definici pole na typ klíče a typ hodnoty `array<klic, hodnota>` ale tak to asi víš :) tak možná nechápu kterou čárku řešíš 👍 ?

Reply:
can u switch to english? I do not understand a word here...

Reply:
Its OK, don't worry.